tests/test_beerapi.py

```diff
@@ -1,28 +1,30 @@
+"""
+    This test is for the purpose of testing beerapi module with pytest.fixture and assuring requests works
```

Comment:
Can you break this up into 2 lines so it conforms to the character count standard